i18n-helpers/src/lib.rs

```diff
@@ -463,7 +463,10 @@ pub fn extract_messages(document: &str) -> Vec<(usize, String)> {
             Group::Translate(events) => {
                 if let Some((lineno, _)) = events.first() {
                     let (text, new_state) = reconstruct_markdown(events, state);
-                    messages.push((*lineno, text));
+                    if !text.trim().is_empty() {
+                        // Skip empty messages.
```

Comment:
Maybe reverse the comment's focus to `// Include messages with content.` I briefly got confused because the test was checking for the presence of content, not its absence.

Reply:
Thanks, that's a good point! I rephrased it to actually explain _why_ we skip the messages, not just _that_ we skip them.